Update the text to reflect new promptql GTM (sales assisted) (#987)
hasura/ddn-docs: docs/ai/overview.mdx

## Introduction

> Contact us to try PromptQL to get AI-powered insights into your data. Read more about PromptQL
> in the dedicated [PromptQL site](https://promptql.hasura.io/).

Reviewer: ```suggestion
[Contact us](https://hasura.io/contact-us) to try PromptQL to get AI-powered insights into your data. Read more about PromptQL
on our [main site](https://hasura.io/).
```
